OAuth configuration steps updated (#1582)
DataDog/integrations-extras: lambdatest/README.md

Here's how you can track incidents in Datadog with LambdaTest:

1. Click "Connect Accounts" to begin authorization of the LambdaTest integration. You will be taken to the Login page in LambdaTest.

> 2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to your Datadog application.

Reviewer: ```suggestion
2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to the Datadog authorization page.
```

3. Click **Authorize** to complete the integration process.
4. A confirmation email is sent once the integration configuration is complete.
5. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.

> 

Reviewer: ```suggestion

### Uninstallation

To ensure this integration is fully uninstalled, revoke authorization in the Configure tab of this tile or in [OAuth Apps](https://app.datadoghq.com/organization-settings/oauth-applications) within Organization Settings in Datadog. 

Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys management page](https://app.datadoghq.com/organization-settings/api-keys).
```

Please also add these OAuth-specific instructions under the Configuration section. Thanks!

Here's how you can track incidents in Datadog with LambdaTest:

1. Click **Connect Accounts** to begin authorization of the LambdaTest integration from the Login page in LambdaTest.

> 2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to the Datadog authorization page.

Reviewer 2: ```suggestion
2. Log in to your LambdaTest account on the LambdaTest website to be redirected to the Datadog authorization page.
```

Reviewer 2: Please remove the future tense here as mentioned in our style guide:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense

> 2. Select settings from the menu bar.
> 3. Click on integrations under settings.

Reviewer: When a user clicks "Connect Accounts" on the datadog tile, they'll be taken to the onboarding_url that you specified when creating the OAuth client, which I believe is this Lambdatest integrations page. If so, we can remove lines 28-30 and replace with the following:

```suggestion
1. Click "Connect Accounts" to begin authorization of the LambdaTest integration. You will be taken to the Integrations page in LambdaTest.
```

> 4. To integrate Datadog, click on install next to Push to Datadog.

Reviewer: ```suggestion
2. In LambdaTest, click "Install" next to **Push to Datadog**. 
```

Author: Updated accordingly

> 5. Click on Authorize button to complete the integration process.

Reviewer: ```suggestion
5. Click **Authorize** to complete the integration process.
```

> 6. A confirmation email is sent. Confirm the integration in the email.

Reviewer: ```suggestion
6. A confirmation email is sent once the integration configuration is complete.
```
